ChangeLog.d/mbedtls_ecp_export.txt

```diff
@@ -1,3 +1,3 @@
 Features
-   * Add mbedtls_ecp_export() function to export ECP
-     keypair parameters. Fixes #4838.
+   * Add a function mbedtls_ecp_export() to export ECP key pair parameters.
```

Comment:
Actually, I think this is better without `a`:

```suggestion
   * Add function mbedtls_ecp_export() to export ECP key pair parameters.
```

Reply:
I agree with Tom that the version without "a" is better out of the two, but I'm not sure if "function" is even needed here, it's pretty self-evident. The addition of details at the end is OK.

Reply:
Why is the version without “a” better? “A” is required for proper English grammar. It's only left out in headlinese.